docs/3.0.0-beta.x/concepts/middlewares.md

```diff
@@ -67,21 +67,19 @@ Every middleware will be injected into the Koa stack. To manage the load order,
 
 ## Configuration and activation
 
-To activate and configure your hook with custom options, you need to edit your `./config/environments/**/middleware.json` file in your Strapi app.
+To configure the middlewares of your application, you need to edit the `./config/middleware.js` file in your Strapi app.
```

Comment:
create and edit as the file is not created by default

Reply:
There is a sentence just under saying the file doesn't exist by default :) will add create or edit to be safe